docker/README.md

```diff
@@ -0,0 +1,108 @@
+# Food Backend
+
+This project is a Spring Boot application with PostgreSQL as the database. Docker is used to containerize both the application and the database, making it easier to run and deploy.
+
+## Prerequisites
+
+- [Docker](https://www.docker.com/)
+- [Docker Compose](https://docs.docker.com/compose/)
+
+## Project Structure
+
+- **Dockerfile**: Defines how to build the Docker image for the Spring Boot application.
+- **docker-compose.yml**: Defines and runs multi-container Docker applications. This file sets up the Spring Boot application and PostgreSQL database.
+- **.env**: Contains environment variables used by the application and PostgreSQL. Make sure this file is correctly configured.
+
+## How to Use
+
+### 1. Start the Application with Docker Compose
+
+To start the application and the PostgreSQL database using Docker Compose, run:
+
+```bash
+docker-compose --file ./docker/docker-compose.yml up -d
+```
+
+This command will:
+
+	•	Build the image (if not already built).
+	•	Start the PostgreSQL database container.
+	•	Start the Spring Boot application container.
+
+### 2. Access the Application
+
+Once the containers are up and running, you can access the application at:
+http://localhost:8080/...
```

Comment:
where does the 8080 come from? 
what if one wants to change it?

Reply:
For change welcome in setting.